### ISSUE TITLE
nested interactive - connection action

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#8829]
### Severity

P3 - Medium (Workaround available but difficult)

### Type of Accessibility Issue

Keyboard Navigation

### Describe the issue

Current behavior: the fast pass failed by running using accessibility insights extension. 
It has nested focus in action UX. see screenshot below

<img width="1388" height="895" alt="Image" src="https://github.com/user-attachments/assets/45fa6bd6-7a24-4860-a533-fddbbc01050f" />

### What type of Logic App are you using?

Consumption (Portal)

### Environment

Browser: Edge

### WCAG Success Criteria (if known)

WCAG 4.1.2
